src/Sfa.Tl.Matching.Web/Startup.cs

```diff
@@ -7,19 +7,33 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.HttpsPolicy;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.WebSockets.Internal;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Sfa.Tl.Matching.Application.Services;
+using Sfa.Tl.Matching.Infrastucture.Configuration;
+using Constants = Sfa.Tl.Matching.Infrastucture.Configuration.Constants;
 
 namespace Sfa.Tl.Matching.Web
 {
     public class Startup
     {
-        public Startup(IConfiguration configuration)
+        public MatchingConfiguration Configuration { get; set; }
+
+        public ILogger<Startup> Logger { get; }
+
+        public Startup(IConfiguration configuration, ILogger<Startup> logger)
```

Comment:
Does this needs to be ILoggerFactory loggerFactory?
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/startup?view=aspnetcore-2.2